pages/opportunities/[slug].tsx

```diff
@@ -69,8 +69,10 @@ const Page: NextPage<PageProps> = (props) => {
             <S.OpportunityContactCard>
               <S.OpportunityMetaRow>
                 <S.OpportunityProjectImg src={parentProject.logoUrl} />
-                {parentProject.silent && <Body>{parentProject.name}</Body>}
-                {!parentProject.silent && (
+                {parentProject.state === "draft" && (
```

Comment:
Nechceme ty `draft` úplně vynechat ze zobrazení?

Reply:
To by bylo super. Momentálně se ale některé ty „draft“ projekty používají u [vypsaných pozic](https://cesko.digital/opportunities), viz například č.edu. Ideální by asi bylo domluvit, abychom projekty pro ty vypsané pozice dotáhli k běžnému zveřejnění a následně nezobrazovat příležitosti spojené s drafty projektů?

Reply:
Ano, to by bylo rozhodně lepší. Tyhle projekty by měly být v nějakém stavu `incubating`?

Reply:
Já vlastně nevím. Původně to proběhlo tak, že jsme u vypsaných příležitostí vyžadovali projekt, takže někdo vyplnil torzo projektů jako č.edu – ale pak všechny překvapilo, že se ten projekt objevil i normálně mezi ostatními projekty ve výpisu na webu, takže se situace dočasně „vyřešila“ tím, že jsou označené jako draft, ačkoliv to tak reálně nefunguje.

Za mě by bylo ideální, kdybys jako PO mohl (1) domluvit jasnější zadání stavu `incubating` (co s projekty v téhle kategorii?) a (2) dotažení těch draftových projektů. Pak můžem přestat zobrazovat příležitosti pro draftové projekty a bude to myslím celé ideál.

Reply:
Říkám si, jestli bychom nemohli tuhle specifickou situaci vyřešit stavem projektu `internal`, přinejmenším dočasně, a postupně to dotáhli?

Reply:
To by šlo. Uměl bys rekapitulovat sémantiku jednotlivých stavů? Tohle je přesně jedna z věcí, které by nebylo marné postupně někde kupit do nějaké „specifikace“.